package.json

```diff
@@ -102,7 +102,8 @@
     "@types/react": "~18.2.0",
     "@types/react-dom": "~18.2.0",
     "@xstate5/react/**/xstate": "^5.19.2",
-    "globby/fast-glob": "^3.3.2"
+    "globby/fast-glob": "^3.3.2",
+    "pkce-challenge": "3.1.0"
```

Comment:
cc @elastic/kibana-operations: Pinning `pkce-challenge` to `3.x` to avoid the ESM issues we're facing with the 4.x version